pkg/api/generate.go

```diff
@@ -20,6 +20,8 @@ func (app *App) Generate(project, branchName string) ([]runtime.Object, error) {
 		"Branch":  branchName,
 	}).Infof("deploying project")
 
+	app.Clients.Statsd.Increment("deployments")
```

Comment:
We would have to configure the exporter: https://github.com/prometheus/statsd_exporter#metric-mapping-and-configuration

Should we do?